cns/fsnotify/fsnotify.go

```diff
@@ -115,6 +118,23 @@ func (w *watcher) watchFS(ctx context.Context) error {
 	if err != nil {
 		w.log.Error("failed to add path to fsnotify watcher", zap.String("path", w.path), zap.Error(err))
 	}
```

Comment:
@camrynl is this error intentionally not returned here? it seems like we shouldn't continue if we fail to `Add`

Reply:
I don't recall, but it would make sense to return here